README.md

```diff
@@ -86,9 +86,10 @@ There are a bunch of options you can use:
 - `inputPagesPath`: The input pages path.
 - `outputPagesPath`: The output pages path.
 
-- `templateStyleguideStylesheet`: The stylesheets to include.
+- `templateStyleguideStylesheet`: The basic stylesheets of the styleguide (can be overwritten, not recommended).
+- `templateStyleguideBrandingStylesheet`: The additional stylesheets to include for customizing the whole theme.
```

Comment:
Pleas do not call it `templateStyleguideBrandingStylesheet`. There are several reasons:

1. Not always useful for branding
2. Can be more than one

For me a better name would be `templateGlobalStylesheets`